magni_demos/launch/fiducial_follow.launch

```diff
@@ -24,10 +24,8 @@
         <param name="adaptiveThreshWinSizeStep" value="8" />
         <param name="adaptiveThreshWinSizeMin" value="10" />
         <param name="adaptiveThreshWinSizeMax" value="50" />
-        <param name="doCornerRefinement" value="False" />
-        <param name="do_tracking" value="false" />
-        <param name="num_threads" value="3" />
```

Comment:
Are you sure these are unused by aruco?

Reply:
do_tracking, num_threads definitely not in aruco_detect.cpp, while the doCornerRefinement was just a typo.

Reply:
Hmm, I've checked some other branches of fiducials and none seem to use it either, I guess we can merge for now.